src/components/button-bar/__snapshots__/button-bar.spec.js.snap

```diff
@@ -0,0 +1,7536 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Button Bar with IconButtons renders an icon correctly 1`] = `
+.c3 {
+  display: inline-block;
+  position: relative;
+  color: rgba(0,0,0,0.65);
+  background-color: transparent;
+  vertical-align: middle;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  display: -webkit-inline-box;
+  display: -webkit-inline-flex;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  -webkit-box-pack: center;
+  -webkit-justify-content: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  height: 24px;
+  width: 24px;
+}
+
+.c3::before {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-family: CarbonIcons;
+  content: "\\e94a";
+  font-size: 16px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 16px;
+  vertical-align: middle;
+  display: block;
+}
+
+.c4:hover .c2 {
+  color: #FFFFFF;
+}
+
+.c4 .c2 {
+  margin-left: 0px;
+  margin-right: 8px;
+  margin-bottom: 0px;
+  height: 16px;
+}
+
+.c4 .c2 svg {
+  margin-top: 0;
+}
+
+.c5:hover .c2 {
+  color: #FFFFFF;
+}
+
+.c5 .c2 {
+  margin-left: 8px;
+  margin-right: 0px;
+  margin-bottom: 0px;
+  height: 16px;
+}
+
+.c5 .c2 svg {
+  margin-top: 0;
+}
+
+.c6:hover .c2 {
+  color: #FFFFFF;
+}
+
+.c6 .c2 {
+  margin-left: 0px;
+  margin-right: 8px;
+  margin-bottom: 0px;
+  height: 16px;
+}
+
+.c6 .c2 svg {
+  margin-top: 0;
+}
+
+.c7:hover .c2 {
+  color: #FFFFFF;
+}
+
+.c7 .c2 {
+  margin-left: 0px;
+  margin-right: 8px;
+  margin-bottom: 0px;
+  height: 16px;
+}
+
+.c7 .c2 svg {
+  margin-top: 0;
+}
+
+.c8:hover .c2 {
+  color: #FFFFFF;
+}
+
+.c8 .c2 {
+  margin-left: 8px;
+  margin-right: 0px;
+  margin-bottom: 0px;
+  height: 16px;
+}
+
+.c8 .c2 svg {
+  margin-top: 0;
+}
+
+.c0 button:not(:first-of-type) {
+  border-left: none;
+}
+
+.c0 button {
+  border: 2px solid #008200;
+}
+
+.c0 button.icon span {
+  color: #008200;
+}
+
+.c0 button.icon span:hover {
+  color: white;
+  background-color: #008200;
+}
+
+.c1 {
+  background: transparent;
+  border: none;
+  padding: 0;
+}
+
+.c1:focus {
+  color: rgba(0,0,0,0.9);
+  background-color: transparent;
+  outline: solid 3px #FFB500;
+  z-index: 1;
+}
+
+.c1:hover {
+  cursor: pointer;
+}
+
+.c1::-moz-focus-inner {
+  border: none;
+}
+
+.c1 .c2 {
+  position: relative;
+}
+
+.c1 .c2:focus {
+  border: none;
+}
+
+<ButtonBar
+  fullWidth={false}
+  iconPosition="before"
+  size="medium"
+>
+  <styled.div
+    fullWidth={false}
+    theme={
+      Object {
```

Comment:
We need to remove these theme objects from the snapshots otherwise anytime the `baseTheme` is changed the snapshots will need updating. If you really need to do a snapshot test TestRenderer will remove the theme

Reply:
Removed snapshots